README.md

```diff
@@ -35,13 +43,13 @@ service account with a few extra roles. This can be enabled with the `-e`
 flag:
 
 ```bash
-./helpers/setup.sh -p PROJECT_ID -o ORG_ID -e
+. ./helpers/setup.sh -p PROJECT_ID -o ORG_ID -e
 ```
 
 Utilizing a shared VPC via a host project is supported with the `-f` flag:
 
 ```bash
-./helpers/setup.sh -p PROJECT_ID -f HOST_PROJECT_ID -o ORG_ID
+. ./helpers/setup.sh -p PROJECT_ID -f HOST_PROJECT_ID -o ORG_ID
```

Comment:
Is the `.` also needed for the cleanup scripts?

Reply:
No, there isn't anything done in the cleanup script that needs to exist after the execution is completed.